tests/integration/test_context_chunker_removal.py

```
@@ -0,0 +1,338 @@
+                line
+                for line in content.splitlines()
+                if line.strip() and not line.strip().startswith("#")
+            ]
+            assert not non_comment_lines, "labeler.yml should be empty or comments only"
+
+    @staticmethod
+    def test_workflow_checks_simplified() -> None:
+        """Workflow checks should be simplified."""
+        apisec_workflow = WORKFLOWS_DIR / "apisec-scan.yml"
+        if apisec_workflow.exists():
+            if "Check for APIsec credentials" in content:
```

Ruff (0.14.14)

[error] Undefined name `content`

(F821)